tests/regression/17-arinc/03-preemt_lock.c

```diff
@@ -1,4 +1,5 @@
-// SKIP PARAM: --set ana.activated[+] extract_arinc --set ana.activated[+] thread --disable ana.arinc.export
+// SKIP PARAM: --set ana.activated[+] arinc --set ana.activated[+] thread --disable ana.arinc.export
+#include <assert.h>
```

Comment:
Maybe add a comment here on why this does not pass such that we don't re-investigate it every year? Or even mark the assertion as TODO?